src/mach/pack/alpha/jib.clj

```diff
@@ -0,0 +1,103 @@
+(ns mach.pack.alpha.jib
+  (:require [mach.pack.alpha.impl.tools-deps :as tools-deps]
+            [clojure.string :as str]
+            [clojure.tools.cli :as cli])
+  (:import (com.google.cloud.tools.jib.api Jib AbsoluteUnixPath)
+           (java.nio.file Paths Files LinkOption)
+           (com.google.cloud.tools.jib.api LayerConfiguration
+                                           Containerizer
+                                           DockerDaemonImage
+                                           TarImage
+                                           RegistryImage
+                                           ImageReference)
+           (com.google.cloud.tools.jib.frontend CredentialRetrieverFactory)))
+
+(def string-array (into-array String []))
+(def target-dir "/home/app")
+
+(defn jib [{::tools-deps/keys [paths lib-map]} {:keys [image-name image-type tar-file base-image target-dir main]}]
+  (println "Building" image-name)
+  (-> (Jib/from base-image)
+      (.addLayer (into []
```

Comment:
I think libraries need to be run through elodin, otherwise:

1. `[foo/core "1.0.0"]`, and `[bar/core "1.0.0"]` both have the filename "core-1.0.0.jar", which means you miss one from the classpath (or get an exception, not sure what jib does)
2. Git dependencies are likely to use `src` as their folder name, this means conflicts again, but also that you have no identification of where they came from.

Reply:
Thank you! Now I understand the need for elodin :)

Reply:
The name is a bit silly, it's a bit of a joke referencing a book, the king killer Chronicle.